jobs/tcp_router/templates/pre-start.erb

```diff
@@ -1,5 +1,4 @@
 #!/bin/bash -e
 
-<% if spec.bootstrap == true %>
+# certs need to be written to the filesystem on all VMs
```

Comment:
I regret saying that we could get rid of this if and have decided I don't like this change. I think this change overloads the config-validator. 

There IS a reason for config validation to only run on 1 VM: the config is the same on every VM. Only 1 VM needs to check the config and fail if something is wrong. There is NOT a reason for cert creation to be run during pre-start. 

Please change the PR so that the certs are created at runtime and not during pre-start.

Reply:
To write during runtime you can do it in the data dir and setup a writeable path via BPM [here](https://github.com/cloudfoundry/routing-release/blob/9956cbcb36fe0c80f404871fbd4782b6655ce414/jobs/tcp_router/templates/bpm.yml.erb#L12-L19). For example "/var/vcap/data/tcp_router/config" is already writable.